src/components/header.ts

```diff
@@ -43,7 +43,10 @@ const getTemplate = () => {
           </li>
           <li class="nav-item" hidden data-header="helpLink">
               <a href="//caleydo.org" target="_blank" class="nav-link" title="${I18nextManager.getInstance().i18n.t('phovea:ui.openHelpPage')}">
-                  <i class="fas fa-question fa-fw" aria-hidden="true"></i>
+                  <span class="fa-stack" style="font-size: 0.5em; height: 2.3em;">
+                    <i class="fas fa-book-open fa-stack-2x"></i>
+                    <i class="fas fa-info fa-stack-1x" style="left: 20%; filter:invert(100%);"></i>
```

Comment:
Could you use the `fa-inverse` class instead of the filter?
And wouldn't it be more scale invariant if we used `em` instead of `%` in the `left` style? That is just a guess though, I am not entirely sure about that.

Reply:
I tried `fa-inverse`, but it always applies `color: #fff` which does not work for our purpose since we toggle the color with the link. Hence, I would need to remove `fa-inverse` or override the color on hover.

The original value from `fa-stack` is also in `%`. That's why I kept the unit and just modified the value. I'm not sure if mixing the units is a good idea. 😉

Reply:
👍